src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index];
+    }
+
+    public int getNumber() {
+        Random random = new Random();
+        int value = random.nextInt(100);
+        return value;
     }
```

Comment:
this method shouldn't belong to the ColorSupplier class because it doesn't relate to colors.

let's generate numbers directly in the getRandomBall method